testing/base/gtest_main.cpp

```diff
@@ -13,5 +14,10 @@ auto main(int argc, char** argv) -> int {
       "https://github.com/carbon-language/carbon-lang/issues and include the "
       "crash backtrace.\n");
   llvm::InitLLVM init_llvm(argc, argv);
+  llvm::InitializeAllTargetInfos();
+  llvm::InitializeAllTargets();
+  llvm::InitializeAllTargetMCs();
+  llvm::InitializeAllAsmParsers();
+  llvm::InitializeAllAsmPrinters();
```

Comment:
You might consider a Lower::Initialize call, to reduce the duplication.

Reply:
I have a pending follow-up cleanup to do that and more, just trying to get this landed first.

Reply:
Merged that in here, since I was making other substantial changes.